test/behavior/tendrl/login.feature

```diff
@@ -12,29 +12,12 @@ Feature: LOGIN
       And User deletes what is in the "password" field
      Then The text "password is required" should appear under the "password" box
 
-  Scenario: Click login with empty fields
-    Given User is at the Tendrl login page
-     When User clicks the Login button
-     Then The “Please fill out this field" popup should appear under the "username" box
-
-  Scenario: Click login button with username but no password
-    Given User is at the Tendrl login page
-     When User types something in the "username" field
-      And User clicks the Login button
-     Then The “Please fill out this field" popup should appear under the "password" box
-
-  Scenario: Click login button with password but no username
-    Given User is at the Tendrl login page
-     When User types something in the "password" field
-      And User clicks the Login button
-     Then The “Please fill out this field" popup should appear under the "username" box
-
   Scenario: Incorrect Username and Password
     Given User is at the Tendrl login page
      When User types in an incorrect "username"
       And User types in an incorrect "password"
       And User clicks the Login button
-     Then The “username or password does not match” should appear above the "username" box
+     Then The “username or password you entered does not match our records” popup should appear above the "username" box
```

Comment:
`“username or password you entered does not match our records”`

Reply:
replace “username or password you entered does not match our records” with "username or password you entered does not match our records"

Reply:
That is the same thing

Reply:
@nathan-weinberg see the difference between the quotes. in your the quotes are `“` and mine are `"`.